packages/liveblocks-core/src/room.ts

```diff
@@ -234,170 +234,186 @@ export type BroadcastOptions = {
   shouldQueueEventIfNotReady: boolean;
 };
 
-export type Room<
+type SubscribeFn<
   TPresence extends JsonObject,
-  TStorage extends LsonObject,
+  _TStorage extends LsonObject,
```

Comment:
I'm curious why this type received an underscore

Reply:
The `TStorage` type isn't used in this definition, and this `_`-prefix avoids a linting error about it. I added the type param despite it not being used, to maintain the same param order as all of our other APIs. This seemed important for future compatibility, because even though we don't explicitly export the `SubscribeFn` type, this type is somewhat part of our public API[^1], and adding in the `TStorage` type param later might result in a breaking change if we did.

[^1]: For example, if you access `Room<P, S, U, E>["subscribe"]`.